src/Workspaces/Remote/Core/xlf/RemoteWorkspacesResources.es.xlf

```diff
@@ -0,0 +1,117 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
+  <file datatype="xml" source-language="en" target-language="es" original="../RemoteWorkspacesResources.resx">
+    <body>
+      <trans-unit id="FeatureName_AddImport">
```

Comment:
preference would be `FeatureName_Add_import`

Reply:
This needs to match the RPC interface name, so that we can look up the feature name based on the interface name.